cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -0,0 +1,617 @@
+---
+title: Provision a CockroachDB Cloud Cluster with Terraform
+summary: Learn how to provision a cluster using the CockroachDB Cloud Terraform provider.
+toc: true
+docs_area: manage
+---
+
+[Terraform](https://terraform.io) is an infrastructure-as-code provisioning tool that uses configuration files to define application and network resources. You can provision CockroachDB Cloud clusters and cluster resources by using the [CockroachDB Cloud Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach) in your Terraform configuration files.
+
+This tutorial shows you how to provision a CockroachDB Cloud cluster using the CockroachDB Cloud Terraform provider.
+
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="serverless"><strong>{{ site.data.products.serverless }}</strong></button>
+    <button class="filter-button page-level" data-scope="dedicated"><strong>{{ site.data.products.dedicated }}</strong></button>
+</div>
+
+## Before you begin
+
+Before you start this tutorial, you must
+
+- [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+- Install the [`wget` command line utility](https://www.gnu.org/software/wget/).
+- Create a [service account](console-access-management.html#service-accounts) and [API key](console-access-management.html#api-access) in the [CockroachDB Cloud Console](https://cockroachlabs.cloud).
+
+## Create the Terraform configuration files
+
+Terraform uses a infrastructure-as-code approach to managing resources. Terraform configuration files allow you to define resources declaratively and let Terraform manage their lifecycle.
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+In this tutorial, you will create a {{ site.data.products.serverless }} cluster with a spend limit of $0.
+
+1. In a terminal create a new directory and use `wget` to download the {{ site.data.products.serverless }} `main.tf` example file.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    wget https://raw.githubusercontent.com/cockroachdb/terraform-provider-cockroach/main/examples/workflows/cockroach_serverless_cluster/main.tf
+    ~~~
+
+1. In a text editor create a new file `terraform.tfvars` in `cockroach_serverless_cluster` with the following settings.
```

Comment:
the directory should be named in the previous step or the name can be removed/generalized here

Reply:
Done.